index.html

```diff
@@ -20,7 +25,22 @@ <h1>Kalender Suvnet -2021- (Blue_group)</h1>
         <p class="dayToday"></p>
         <p class="date"></p>
       </div>
-      <div class="todoContainer"></div>
+      <div class="todoContainer">
+        <div class="todoheader">
+          <h4>Lägg till ny aktivitet</h4>
+          <i id="addTodo-button" class="material-icons addTodo-button"
+            >add_box</i
+          >
+        </div>
+        <div id="input-block" class="input-block">
+          Titel:<br />
+          <input id="title" type="text" /><br />
+          Beskrivning:<textarea id="description" rows="10" cols="30"></textarea
+          ><br />
+          Datum:<br /><input id="date" type="date" />
+          <button id="createTodo-button">Skapa</button>
+        </div>
```

Comment:
Istället för en massa < br />, ändra input-block till flex-column och ändra input titlarna till < label for="input id" >input titel< /label >

Reply:
Bra feedback, ändrade de som du skrev ser bättre ut nu :) Men fick ha med några <br> ändå annars såg de konstigt ut

Reply:
< br />